helpers/createVerificationToken.js

```diff
@@ -0,0 +1,3 @@
+const uuid = require("uuid");
+
+module.exports = () => uuid.v4();
```

Comment:
It makes no sense to me to create function like that. Better to call uuid directly